src/client/components/TestComponent/Reservations.js

```diff
@@ -0,0 +1,24 @@
+import React from "react";
+import {useState,useEffect} from "react"
+
+export default function Reservations(){
```

Comment:
Is this component used? If not, try to delete files/components that you're not using